data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/test_overwrite_specials_filter.cfg

```diff
@@ -0,0 +1,110 @@
+#####
+# API(s) being tested: [abilities]['weapon']overwrite_specials filter [overwrite_filter]
```

Comment:
It looks like this would more accurately be:
`[damage]overwrite_specials=both_sides,[overwrite_filter]`

Reply:
corrected